Makefile

```diff
@@ -193,9 +197,7 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 
 .install.md2man: .gopathok
 	if [ ! -x "$(GOPATH)/bin/go-md2man" ]; then \
-		   go get -d github.com/cpuguy83/go-md2man; \
-		   cd $(GOPATH)/src/github.com/cpuguy83/go-md2man; \
-		   git checkout 20f5889cbdc3c73dbd2862796665e7c465ade7d1; \
+		   go get -u github.com/cpuguy83/go-md2man; \
 		   go install github.com/cpuguy83/go-md2man; \
```

Comment:
Can remove this too

Reply:
does -u imply install

Reply:
Yeah. -u will install if not present, update if present